<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetectionWindow</class>
 <widget class="QDialog" name="DetectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detection</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color:  #2a292e;
}

QLabel {
color: white;
font-size: 18px;
font-weight: bold;
}

QPushButton{
font-size: 18px;
font-weight: bold;
color: #1db954;
border: 2px solid #1db954;
border-radius: 14px;
letter-spacing: 1px;
}
QPushButton:hover {
color: #121212;
background-color: 	#1db954;
border: 2px solid #121212;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>25</number>
     </property>
     <property name="rightMargin">
      <number>25</number>
     </property>
     <property name="bottomMargin">
      <number>25</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="detected_gesture_name">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="detection_confidence">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="detection_camera">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>56</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>56</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>START</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
